src/gravel/tests/conftest.py

```diff
@@ -311,6 +286,11 @@ class FakeStorage(Storage):  # type: ignore
             total = 2000  # type: ignore
 
         gstate: GlobalState = GlobalState()
+        # gstate's KV store will start a thread to try to connect to the
```

Comment:
A better fix here is to fake the connection method.

Reply:
How do I do that? when gstate is constructed it creates a real KV (which starts the connector thread), then later we replace that with a FakeKV (`gstate._kvstore = FakeKV()`). I could fake the connection method in FakeKV, but by then it's too late.

Reply:
Oooh, I just had a thought - if I change the GlobalState constructor to take a KV as a param (which defaults to KV) then I should be able to override that in the test with a FakeKV.